spec/models/orchestration_stack_retire_task_spec.rb

```diff
@@ -0,0 +1,38 @@
+describe OrchestrationStackRetireTask do
+  let(:user) { FactoryGirl.create(:user_with_group) }
+  let(:orchestration_stack) { FactoryGirl.create(:orchestration_stack) }
+  let(:miq_request) { FactoryGirl.create(:orchestration_stack_retire_request, :requester => user) }
+  let(:miq_request_task) { FactoryGirl.create(:miq_request_task, :miq_request_id => miq_request.id) }
+  let(:orchestration_stack_retire_task) { FactoryGirl.create(:orchestration_stack_retire_task, :source => orchestration_stack, :miq_request_task_id => miq_request_task.id, :miq_request_id => miq_request.id, :options => {:src_ids => [orchestration_stack.id] }) }
+  let(:reason) { "Why Not?" }
+  let(:approver) { FactoryGirl.create(:user_miq_request_approver) }
+  let(:zone) { FactoryGirl.create(:zone, :name => "fred") }
+
+  it "should initialize properly" do
+    expect(orchestration_stack_retire_task.state).to eq('pending')
```

Comment:
`expect(orchestration_stack_retire_task).to have_attributes(:state => "pending", :status => "ok")`?